src/util/audiosignal.cpp

```diff
@@ -4,6 +4,31 @@
 
 namespace mixxx {
 
+// Definitions of static class constants are required by
+// LLVM CLang, but cause "multiply defined symbols" errors
+// in Visual Studio 2013. GCC is happy either with or
```

Comment:
I'm confused -- how does this build at head (since Travis/Appveyor uses all of GCC, LLVM, and MSVC2013)?